languages/python/oso/tests/test_polar_roles.py

```diff
@@ -401,30 +310,7 @@ def test_relationship_without_resources(init_oso):
     oso.load_str(policy)
 
     with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
-
-
-@pytest.mark.skip("TODO: validation")
-def test_duplicate_role_name_same_resource(init_oso):
-    oso, session = init_oso
-    policy = """
-    resource(_type: Org, "org", actions, roles) if
-        actions = [
-            "invite", "create_repo"
-        ] and
-        roles = {
-            owner: {
-                permissions: ["invite"],
-                implies: ["member", "repo:member"]
-            },
-            owner: {
-                permissions: ["create_repo"]
-            }
-        };
-        """
-    oso.load_str(policy)
-    with pytest.raises(OsoError):
-        oso.roles.synchronize_data()
```

Comment:
What happens if you have a pair of `resource/4` predicates that define the same role?

Reply:
Role names aren't unique across resource types so that's fine.

Reply:
I mean a pair of `resource/4` predicates for the same resource type that define the same role